libs/dal/src/repositories/subscriber/subscriber.schema.ts

```diff
@@ -193,34 +193,13 @@ subscriberSchema.index({
   deleted: 1,
 });
 
-subscriberSchema.index({
-  _environmentId: 1,
-  _organizationId: 1,
-  createdAt: 1,
-  _id: 1,
-});
```

Comment:
if we sort by `_id` we will still need index for it.
something like 

```
subscriberSchema.index({
  _environmentId: 1,
  _organizationId: 1,
  _id: 1,
});
```

id suggest to check if we need 

```
subscriberSchema.index({
  _environmentId: 1,
  _organizationId: 1,
  deleted: 1,
});
```

if we do not we could remove it and add the suggested above instead.